unittests/protocol_feature_tests.cpp

```diff
@@ -1074,13 +1074,9 @@ BOOST_AUTO_TEST_CASE( get_sender_test ) { try {
 BOOST_AUTO_TEST_CASE( protocol_activatation_works_after_transition_to_savanna ) { try {
    validating_tester c({}, {}, setup_policy::preactivate_feature_and_new_bios );
 
-   const auto& pfm = c.control->get_protocol_feature_manager();
-   const auto& d = pfm.get_builtin_digest(builtin_protocol_feature_t::instant_finality);
-   // needed for bios contract
-   const auto& dp = pfm.get_builtin_digest(builtin_protocol_feature_t::bls_primitives);
-   const auto& dw = pfm.get_builtin_digest(builtin_protocol_feature_t::wtmsig_block_signatures);
-   const auto& dwk = pfm.get_builtin_digest(builtin_protocol_feature_t::webauthn_key);
-   c.preactivate_protocol_features( {*d, *dp, *dw, *dwk} );
+      // needed for bios contract
+      // const auto& dwk = pfm.get_builtin_digest(builtin_protocol_feature_t::webauthn_key);
```

Comment:
Remove commented out code and fix indent.

Reply:
Thanks, done.